tmt/utils.py

```diff
@@ -871,7 +994,14 @@ def dict_to_yaml(data, width=None, sort=False, start=False):
     return output.getvalue()
 
 
-def yaml_to_dict(data, check_version=False, yaml_type=None):
+YamlTypType = Literal['rt', 'safe', 'unsafe', 'base']
+
+
+def yaml_to_dict(data: Any,
+                 check_version: bool = False,
+                 yaml_type: Optional[YamlTypType] = None) -> Tuple[bool,
```

Comment:
The indentation/line breaks on the returning type look a bit weird to me here but that's probably a matter of personal preference for line breaks.

Reply:
It's absolutely and undeniably ugly :) FTR, I *did not* format the line in this way.

I guess it could be avoided, to some extent, by asking autopep8 to allow longer lines, but that's just a temporary solution, there will be functions whose signature won't fit into 120 characters, for example. Or by using a different code formatter, but Black will butcher other lines for sure. Who knows what's better...